solutions.js

```diff
@@ -1,5 +1,51 @@
 console.log('Defining functions!');
 
+
 function changeNavColor() {
   // change the navbar to a different color
-};
+  var navTop = document.querySelector('#topbar');
+  navTop.style.backgroundColor = 'green';
+}
+
+function addTopHeading() {
+  var createElement = document.createElement('h1');
+  createElement.setAttribute('class', 'customClass');
+  createElement.innerHTML = 'This page is now under Matt\'s control!';
+  document.getElementById('content').prepend(createElement);
+  createElement.style.fontSize = '50px';
+  createElement.style.color = 'blue';
+}
+
+function removeHalf() {
+  var links = [...document.querySelectorAll('.cards .image-list-link')];
+  var posts = [...document.querySelectorAll('.cards .post')];
+  var linksLength = links.length / 2;
+  for (let i = 0; i < linksLength; i++) {
+   links[i].parentNode.removeChild(links[i]);
+   posts[i].style.display = 'none';
+  }
+}
+
+function alertAfterImgClick() {
+  var selectImgs = [...document.querySelectorAll('.cards .image-list-link')];
+  
+  selectImgs.forEach((link) => {
+    link.addEventListener('click', (e) => {
+      e.preventDefault();
```

Comment:
You could have also moved this into the condition and then you wouldn't need `window.open`.

Reply:
ahhhh